crates/bevy_text/src/text_access.rs

```diff
@@ -281,15 +303,15 @@ impl<'w, 's, R: TextRoot> TextWriter<'w, 's, R> {
     /// Gets the [`TextStyle`] of a text span within a text block at a specific index in the flattened span list.
     ///
     /// Panics if there is no span at the requested index.
-    pub fn style(&mut self, root_entity: Entity, index: usize) -> Mut<TextStyle> {
+    pub fn style(&mut self, root_entity: Entity, index: usize) -> Mut<TextFont> {
```

Comment:
Needs to be renamed :)

Reply:
and similar methods added for `TextColor` I assume?